apps/nextjs-website/react-components/types/PreFooter/PreFooter.ts

```diff
@@ -0,0 +1,20 @@
+import { CommonProps, Generic } from '../common/Common.types';
+
+type ImgProps = React.DetailedHTMLProps<
+  React.ImgHTMLAttributes<HTMLImageElement>,
+  HTMLImageElement
+>;
+
+export interface StoreButtonsProps {
+  readonly hrefGoogle?: string;
+  readonly hrefApple?: string;
+}
```

Comment:
Is there a reason to have these be optional since `storeButtons` below is already optional?

The only one I can see is to allow only one storeButton to be rendered, but is that a possible case we want to allow?
I don't think it's realistic for a PagoPA product to be on only one store right?